tpm2/structures.go

```diff
@@ -1023,8 +1025,58 @@ type AuthCommand struct {
 
 // TPMLDigest represents the TPML_Digest structure
 // It is used to convey a list of digest values.
-//This type is used in TPM2_PolicyOR() and in TPM2_PCR_Read()
+// This type is used in TPM2_PolicyOR() and in TPM2_PCR_Read()
 type TPMLDigest struct {
-	Count   uint32
 	Digests []tpmutil.U16Bytes
 }
+
+func (list *TPMLDigest) String() string {
+	var s strings.Builder
+	s.WriteString("TPMLDigest{")
+	for iterator, item := range list.Digests {
+		s.WriteString("Digest")
+		s.WriteString(string(iterator) + "{")
+		s.WriteString(string(item) + "\n")
```

Comment:
`item` is likely to contain non-printable characters - did you mean to encode each digest to something printable (like hexadecimal) here?

Reply:
I got rid of the String function